packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/SwapOrderTx/model.tsx

```diff
@@ -9,6 +10,15 @@ import {
 } from '../components'
 import { Text } from 'blockchain-info-components'
 
+const getOutputCoinDisplayName = (props: Props) => {
+  return path(
+    [`${getOutput(props.order)}`, 'displayName'],
+    props.supportedCoins
+  )
+}
```

Comment:
this is same function as `getSymbolDisplayName` might be good to move it to some more common place and just reuse

Reply:
yeah i would put the shared function in `packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/model.tsx` and then import into the 3 templates that need it